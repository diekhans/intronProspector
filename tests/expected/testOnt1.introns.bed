chr1	973640	973832	sj0	1	.	973640	973832	64,128,64
chr1	974053	974317	sj1	1	.	974053	974317	64,128,64
chr1	974366	974443	sj2	1	.	974366	974443	64,128,64
chr1	1645262	1649498	sj3	1	.	1645262	1649498	64,128,64
