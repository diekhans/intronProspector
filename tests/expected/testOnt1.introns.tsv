chrom	intron_start	intron_end	strand	uniq_mapped_count	multi_mapped_count	unsure_mapped_count	max_left_overhang	max_right_overhang	confidence	splice_sites	name
chr1	973640	973832	.	0	0	1	62	134	0		sj0
chr1	974051	974315	.	0	0	1	85	49	0		sj1
chr1	1645262	1649498	.	0	1	0	86	53	0		sj2
